tpot/gp_deap.py

```diff
@@ -103,7 +103,8 @@ def varOr(population, toolbox, lambda_, cxpb, mutpb):
 
 
 def eaMuPlusLambda(population, toolbox, mu, lambda_, cxpb, mutpb, ngen, pbar,
-                   stats=None, halloffame=None, verbose=0, max_time_mins=None):
+                   stats=None, halloffame=None, verbose=0, max_time_mins=None,
+                   periodic_pipeline_saver=None):
```

Comment:
actually hook seems a bit confusing to some people.
ill just call it per_generation_function